identity/service.go

```diff
@@ -351,9 +351,7 @@ func (s *Service) propagateDataHandler(msg network.Message) {
 		case *ProposeSend:
 			p := msg.(*ProposeSend)
 			sid.Proposed = p.Propose
-			if len(sid.Proposed.Votes) == 0 {
-				sid.Proposed.Votes = map[string]*crypto.SchnorrSig{}
-			}
+			sid.Proposed.Votes = make(map[string]*crypto.SchnorrSig)
```

Comment:
Dang - now I'm not sure anymore. It could be advantageous for the client to send the proposition directly with his vote - one less message. I erroneously thought we need to ensure no signatures on proposition `n` validate a proposition `n+1`, but that cannot happen, as it's signatures. I'll change it back to the if, as there is no attack possible.